specification/cognitiveservices/data-plane/Face/readme.md

```diff
@@ -0,0 +1,37 @@
+# Cognitive Services Face SDK
+
+> see https://aka.ms/autorest
+
+Example configuration for generating Face SDK.
+
+The current release is `release_1_0`.
+
+``` yaml
+
+tag: release_1_0
+```
+# Releases
+
+### Release 1.0
+These settings apply only when `--tag=release_1_0` is specified on the command line.
+
+``` yaml $(tag) == 'release_1_0'
+input-file: v1.0/Face.json
+
+```
+
+## CSharp Settings
+These settings apply only when `--csharp` is specified on the command line.
+``` yaml $(csharp) 
+csharp: 
+  namespace: Microsoft.CognitiveServices.Face
+
+```
+## NodeJS Settings
+
```

Comment:
please remove this config :) we don't use namespaces in our nodejs sdks and the setting's description also says it will be applied for `--csharp` :)